docker/docker-compose.yml

```diff
@@ -5,7 +5,9 @@ volumes:
 
```

Comment:
It is no point to move this file, unless we have serval projects in one repo

Reply:
Ok I think you're right. Sorry for messing things up! Great you're here 👍